pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -67,909 +90,861 @@
+    "QubitSum",
+    "OrbitalRotation",
+    "QFT",
+    "ECR",
+    "BlockEncode",
+}
+
+allowed_observables = {
+    "PauliX",
+    "PauliY",
+    "PauliZ",
+    "Hadamard",
+    "SparseHamiltonian",
+    "Hamiltonian",
+    "LinearCombination",
+    "Hermitian",
+    "Identity",
+    "Sum",
+    "Prod",
+    "SProd",
+}
+
+gate_cache_needs_hash = (
+    qml.BlockEncode,
+    qml.ControlledQubitUnitary,
+    qml.DiagonalQubitUnitary,
+    qml.MultiControlledX,
+    qml.OrbitalRotation,
+    qml.PSWAP,
+    qml.QubitUnitary,
+)
+
+
+class LightningGPU(LightningBase):  # pylint: disable=too-many-instance-attributes
+    """PennyLane Lightning GPU device.
+
+    A GPU-backed Lightning device using NVIDIA cuQuantum SDK.
+
+    Use of this device requires pre-built binaries or compilation from source. Check out the
+    :doc:`/lightning_gpu/installation` guide for more details.
+
+    Args:
+        wires (int): the number of wires to initialize the device with
+        mpi (bool): enable MPI support. MPI support will be enabled if ``mpi`` is set as``True``.
+        mpi_buf_size (int): size of GPU memory (in MiB) set for MPI operation and its default value is 64 MiB.
+        sync (bool): immediately sync with host-sv after applying operations
+        c_dtype: Datatypes for statevector representation. Must be one of ``np.complex64`` or ``np.complex128``.
+        shots (int): How many times the circuit should be evaluated (or sampled) to estimate
+            the expectation values. Defaults to ``None`` if not specified. Setting
+            to ``None`` results in computing statistics like expectation values and
+            variances analytically.
+        batch_obs (Union[bool, int]): determine whether to use multiple GPUs within the same node or not
+    """
+
+    name = "Lightning GPU PennyLane plugin"
+    short_name = "lightning.gpu"
+
+    operations = allowed_operations
+    observables = allowed_observables
+    _backend_info = backend_info
+    config = Path(__file__).parent / "lightning_gpu.toml"
+    _CPP_BINARY_AVAILABLE = LGPU_CPP_BINARY_AVAILABLE
+
+    def __init__(
+        self,
+        wires,
+        *,
+        mpi: bool = False,
+        mpi_buf_size: int = 0,
+        sync=False,
+        c_dtype=np.complex128,
+        shots=None,
+        batch_obs: Union[bool, int] = False,
+    ):  # pylint: disable=too-many-arguments
+        if c_dtype is np.complex64:
+            self.use_csingle = True
+        elif c_dtype is np.complex128:
+            self.use_csingle = False
+        else:
+            raise TypeError(f"Unsupported complex type: {c_dtype}")
+
+        super().__init__(wires, shots=shots, c_dtype=c_dtype)
+
+        self._dp = DevPool()
+
+        if not mpi:
+            self._mpi = False
+            self._num_local_wires = self.num_wires
+            self._gpu_state = _gpu_dtype(c_dtype)(self._num_local_wires)
+        else:
+            self._mpi = True
+            self._mpi_init_helper(self.num_wires)
```

Comment:
This is normal, it will arise whenever we merge a PR with `ci:use-multi-gpu-runner` and then have a PR without it according to my observation. But I'll add the label and rerun.